src/client/app/components/UIOptionsComponent.jsx

```diff
@@ -81,24 +113,32 @@ export default class UIOptionsComponent extends React.Component {
 		const divPadding = {
 			paddingTop: '35px'
 		};
+		const divBottomPadding = {
+			paddingBottom: '15px'
+		};
+		const multiSelectStyle = {
+			maxWidth: '100%',
+		};
+
+		// Construct the options of the MultiSelect. Because value can be any JavaScript object, here we load it with both the type
+		// and ID. Currently this is useless, but when groups graphing is introduced it will be important
+		const selectOptions = this.props.meters.map(meter => (
+			{ 	label: meter.name,
+				type: DATA_TYPE_METER,
+				value: meter.id,
+			}
+		));
 		return (
 			<div className="col-xs-2" style={divPadding}>
 				<div className="col-xs-11">
-					<div>
-						<div className="form-group">
-							<p style={labelStyle}>Select meters:</p>
-							<select multiple className="form-control" id="meterList" size="8" onChange={this.handleMeterSelect}>
-								{this.props.meters.map(meter =>
-									<option key={meter.id} value={meter.id}>{meter.name}</option>
-								)}
-							</select>
-						</div>
+					<p style={labelStyle}>Meters:</p>
+					<div style={divBottomPadding}>
+						<MultiSelect options={selectOptions} placeholder="Select Meters" theme="bootstrap3" style={multiSelectStyle} onValuesChange={this.handleDatasourceSelect} />
 					</div>
 					<p style={labelStyle}>Graph Type:</p>
 					<div className="radio">
 						<label><input type="radio" name="chartTypes" value={chartTypes.line} onChange={this.handleChangeChartType} checked={this.props.chartToRender === chartTypes.line} />Line</label>
-					</div>
-					<div className="radio">
+						&nbsp; &nbsp; { /* For alignment. */ }
```

Comment:
Don't use this for alignment reasons. Use something else like CSS's inline-block.

Reply:
Done